doc/tutorial/climada_engine_unsequa.ipynb

```diff
@@ -2480,6 +2481,440 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2024-01-18 10:41:36,994 - climada.engine.unsequa.calc_base - WARNING - \n",
+      "\n",
+      "The input parameter x_exp is shared among at least 2 input variables. Their uncertainty is thus computed with the same samples for this input paramter.\n",
+      "\n",
+      "\n",
+      "2024-01-18 10:41:36,998 - climada.engine.unsequa.calc_base - WARNING - \n",
+      "\n",
+      "The input parameter G is shared among at least 2 input variables. Their uncertainty is thus computed with the same samples for this input paramter.\n",
+      "\n",
+      "\n",
+      "2024-01-18 10:41:36,999 - climada.engine.unsequa.calc_base - WARNING - \n",
+      "\n",
+      "The input parameter v_half is shared among at least 2 input variables. Their uncertainty is thus computed with the same samples for this input paramter.\n",
+      "\n",
+      "\n",
+      "2024-01-18 10:41:37,000 - climada.engine.unsequa.calc_base - WARNING - \n",
+      "\n",
+      "The input parameter vmin is shared among at least 2 input variables. Their uncertainty is thus computed with the same samples for this input paramter.\n",
+      "\n",
+      "\n",
+      "2024-01-18 10:41:37,002 - climada.engine.unsequa.calc_base - WARNING - \n",
+      "\n",
+      "The input parameter k is shared among at least 2 input variables. Their uncertainty is thus computed with the same samples for this input paramter.\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from climada.engine.unsequa import CalcDeltaImpact\n",
+    "calc_imp = CalcDeltaImpact(exp_iv, impf_iv, haz, \n",
+    "                           exp_iv, impf_iv, haz_fut_iv)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2024-01-18 10:41:46,609 - climada.engine.unsequa.calc_base - INFO - Effective number of made samples: 1792\n",
+      "2024-01-18 10:41:46,633 - climada.engine.unsequa.calc_base - INFO - \n",
+      "\n",
+      "Estimated computaion time: 0:00:40.678400\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "output_imp = calc_imp.make_sample(N=2**7)\n",
+    "output_imp.get_samples_df().tail()\n",
+    "\n",
+    "output_imp = calc_imp.uncertainty(output_imp)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plotting functionalities work analogous to CalcImpact. However, the figure labelling is wrong. What we calculate and plot is the relative risk change in % and not an absolute number in USD!!!"
```

Comment:
Can this be fixed? 🤔

Reply:
yes. I added a new argument (`calc_delta`) in the plotting functions, which allows you to print out x-axis labels correctly.